sdk/storage/azblob/README.md

```diff
@@ -12,18 +12,40 @@ The Azure Blob SDK can access an Azure Storage account.
 ### Prerequisites
 
 * Go versions 1.16 or higher
-* You must have an [Azure storage account][azure_storage_account]
+* You must have an [Azure storage account][azure_storage_account]. If you need to create one, you can use the [Azure Cloud Shell](https://shell.azure.com/bash) to create one with these commands (replace `"my-resource-group"` and `"myStorageAccount"` with your own unique names):
```

Comment:
Lowercase letters only for storage account `myStorageAccount`. Can you also validate nomenclature for resource group as well?